tasks/shared_libraries/run_tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+## =============================================================================
+## NAME: run_tests.sh
+##
+##
+## DESCRIPTION
+## -----------------------------------------------------------------------------
+## Tests the shared library for SQS queue using unit tests.
+##
+##
+## USAGE
+## -----------------------------------------------------------------------------
+## shared_libraries/run_tests.sh
+##
+## This must be called from the directory /tasks/shared_libraries
+## =============================================================================
+
+## Set this for Debugging only
+#set -ex
+
+## Make sure we are calling the script the correct way.
+BASEDIR=$(dirname $0)
+if [ "$BASEDIR" != "." ]; then
```

Comment:
Base directory should be `bin` not `.` in order to accommodate bamboo scripts. The expectation is you would run this from the codes base directory so if you were in `tasks/shared_libraries/recovery` You would call the script via `bin/run_tests.sh`